addons/sale_timesheet/tests/test_project.py

```diff
@@ -175,6 +175,8 @@ def test_analytic_account_balance(self):
             ],
         })
 
+        self.project_global.invalidate_recordset()
+        self.project_global.analytic_account_id.invalidate_recordset()
```

Comment:
mmh why do you need to manually invalidate the recordset? :thinking:

Reply:
Because I can't put the right dependence on `line_ids` (because of the dynamic fields), so it is not automatically invalidated

Reply:
mmh mwai... it ain't an issue 'till it is an issue, I suppose

Reply:
If it really becomes an issue, we can invalidate manually when writing on a dynamic field of `account.analytic.line`, but I really think it's overkill